pygeoapi/pygeoapi.config.yml

```diff
@@ -319,6 +319,43 @@ resources:
               id_field: id
               table: ref_gages
               uri_field: uri
+    mainstems:
+        type: collection
+        title: Reference Mainstems
+        geojsonld: false
+        description: US Reference Mainstem Rivers
+        keywords:
+            - Mainstem
+        context:
+            - schema: https://schema.org/
+              type: "@type"
+              name_at_outlet: schema:name
+              head_nhdpv2_COMID: https://www.wikidata.org/wiki/Q7376362
```

Comment:
Is this at all the right way to do this?

Reply:
you're going to want to do something like:

```
- schema: https://schema.org/
  name_at_outlet: schema:name
- wiki: https://www.wikidata.org/wiki/
  head_nhdpv2_COMID: 
     "@id": wiki:Q7376362
     "@type": "@id"

```

Reply:
any predicate whose object you want to be a URI you have to specify in that dumb nested way where it's "@type" (object type) is "@id"

Reply:
Poor `@type` and `@id` users... lol 

I'll fix this up now.